views/user/authorized_index.ejs

```diff
@@ -3887,6 +3887,8 @@
             }
             let startDate = Infinity;
             let endDate = 0;
+            let maxDate = Infinity;
+            let minDate = Infinity;
```

Comment:
These declarations are still only in the scope of `renderOverviewChart()`. Also, can you explain your rationale for initializing these to `Infinity`?